xapi-ingest-microservice/lib/xapi.js

```diff
@@ -150,9 +150,8 @@ var saveStatement = module.exports.saveStatement = function(ctx, statement, call
                 console.log({'err': err}, 'Failed to store a new learning activity statement');
                 return callback({'code': 500, 'msg': err.message});
               }
-
-              console.log({'statement': statement}, 'Successfully stored learning activity statement');
-              return callback();
+              console.log({'statement': JSON.stringify(statement)}, 'Successfully stored learning activity statement');
```

Comment:
Yes. It does get a little verbose. But, it helps to have it since Lambda logs can get cryptic at times when the process errors out.